tron/commands/cmd_utils.py

```diff
@@ -40,18 +40,26 @@ class ExitCode(object):
 opener = open
 
 
-def build_option_parser(usage, parser_class=optparse.OptionParser):
-    parser = parser_class(usage, version="%%prog %s" % tron.__version__)
+def build_option_parser(usage=None, parser_class=argparse.ArgumentParser, epilog=None):
```

Comment:
I can't see any reason to keep the parser_class argument around - just bin it and make everything use argparse.ArgumentParser.